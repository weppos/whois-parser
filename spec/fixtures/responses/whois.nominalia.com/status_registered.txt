Domain Name: NOMINALIA.COM
Registry Domain ID: 4546874_DOMAIN_COM-VRSN
Registrar WHOIS Server: whois.nominalia.com
Registrar URL: http://www.nominalia.com
Updated Date: 2013-06-27T00:00:00Z
Creation Date: 2008-02-07T00:00:00Z
Registrar Registration Expiration Date: 2018-11-30T00:00:00Z
Registrar: NOMINALIA INTERNET S.L.
Registrar IANA ID: 76
Registrar Abuse Contact Email: abuse@nominalia.com
Registrar Abuse Contact Phone: +34.935074387
Reseller:
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Registry Registrant ID:
Registrant Name: Nominalia Internet S.L.
Registrant Organization: Nominalia Internet S.L.
Registrant Street: Josep Pla, 2. Torres Diagonal Litoral, B3, 3D
Registrant City: Barcelona
Registrant State/Province: B
Registrant Postal Code: ES-08019
Registrant Country: ES
Registrant Phone: +34.935074360
Registrant Phone Ext:
Registrant Fax: +34.933102360
Registrant Fax Ext:
Registrant Email: gestiontld@nominalia.com
Registry Admin ID:
Admin Name: Claudio Corbetta
Admin Organization: Nominalia Internet S.L.
Admin Street: Josep Pla, 2. Torres Diagonal Litoral, B3, 3D
Admin City: Barcelona
Admin State/Province: B
Admin Postal Code: 08019
Admin Country: ES
Admin Phone: +34.935074360
Admin Phone Ext:
Admin Fax: +34.933102360
Admin Fax Ext:
Admin Email: gestiontld@nominalia.com
Registry Tech ID:
Tech Name: Billing Department
Tech Organization: Nominalia Internet, S.L.
Tech Street: Josep Pla, 2. Torres Diagonal Litoral. Edificio B3, planta 3-D
Tech City: Barcelona
Tech State/Province: Barcelona
Tech Postal Code: 08019
Tech Country: ES
Tech Phone: +34.935074360
Tech Phone Ext:
Tech Fax: +34.933102360
Tech Fax Ext:
Tech Email: payment@nominalia.com
Name Server: DNS1.NOMINALIA.COM
Name Server: DNS2.NOMINALIA.COM
DNSSEC: unsigned
URL of the ICANN WHOIS Data Problem Reporting System: http://wdprs.internic.net/
>>> Last update of whois database: 2018-05-03T10:01:43Z <<<

For more information on Whois status codes, please visit https://icann.org/epp
