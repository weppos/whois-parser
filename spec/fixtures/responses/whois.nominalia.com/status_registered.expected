#domain
  %s == "nominalia.com"

#domain_id
  %s == "4546874_DOMAIN_COM-VRSN"

#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true

#created_on
  %s %CLASS{time}
  %s %TIME{2008-02-07 00:00:00 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2013-06-27 00:00:00 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2018-11-30 00:00:00 UTC}

#registrar
  %s %CLASS{registrar}
  %s.id           == "76"
  %s.name         == "NOMINALIA INTERNET S.L."
  %s.organization == "NOMINALIA INTERNET S.L."
  %s.url          == "http://www.nominalia.com"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].name          == "Nominalia Internet S.L."
  %s[0].organization  == "Nominalia Internet S.L."
  %s[0].address       == "Josep Pla, 2. Torres Diagonal Litoral, B3, 3D"
  %s[0].city          == "Barcelona"
  %s[0].zip           == "L3R 8B5"
  %s[0].state         == "ON"
  %s[0].country_code  == "ES"
  %s[0].phone         == "+34.935074360"
  %s[0].fax           == "+34.933102360"
  %s[0].email         == "gestiontld@nominalia.com"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].name          == "Claudio Corbetta"
  %s[0].organization  == "Nominalia Internet S.L."
  %s[0].address       == "Josep Pla, 2. Torres Diagonal Litoral, B3, 3D"
  %s[0].city          == "Barcelona"
  %s[0].zip           == "L3R 8B5"
  %s[0].state         == "ON"
  %s[0].country_code  == "ES"
  %s[0].phone         == "+34.935074360"
  %s[0].fax           == "+34.933102360"
  %s[0].email         == "gestiontld@nominalia.com"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].name          == "Billing Department"
  %s[0].organization  == "Nominalia Internet, S.L."
  %s[0].address       == "Josep Pla, 2. Torres Diagonal Litoral. Edificio B3, planta 3-D"
  %s[0].city          == "Barcelona"
  %s[0].zip           == "L3R 8B5"
  %s[0].state         == "ON"
  %s[0].country_code  == "ES"
  %s[0].phone         == "+34.935074360"
  %s[0].fax           == "+34.933102360"
  %s[0].email         == "gestiontld@nominalia.com"

#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "dns1.nameresolvers.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "dns2.nameresolvers.com"
